packages/jupyterlab-manager/src/manager.ts

```diff
@@ -168,9 +176,13 @@ class WidgetManager extends ManagerBase<Widget> implements IDisposable {
   /**
    * Restore widgets from kernel and saved state.
    */
-  async restoreWidgets(notebook: INotebookModel): Promise<void> {
-    await this._loadFromKernel();
-    await this._loadFromNotebook(notebook);
+  async restoreWidgets(notebook: INotebookModel, {loadKernel, loadNotebook} = {loadKernel: true, loadNotebook: true}): Promise<void> {
```

Comment:
So something like `options = {loadKernel: true, loadNotebook: true}`? I don't think that's significantly easier to read, but maybe that's because I've been using unpacking more and more and sort of like it.